fusio/src/impls/disk/tokio_uring/fs.rs

```diff
@@ -22,7 +22,7 @@ impl Fs for TokioUringFs {
             .write(options.write.is_some())
             .create(options.create)
             .append(options.write == Some(WriteMode::Append))
-            .truncate(options.write == Some(WriteMode::Overwrite))
+            .truncate(options.truncate)
```

Comment:
Are there any reasons for separate truncate semantic from `WriteMode::Overwrite`?